### ISSUE TITLE
Unable to build deployment package within Cloud or locally (versioned)

### ISSUE THREAD

**Issue by ArjenLammers** [cdcharlebois/mx-crashlytics#1]
Whenever I try to build a project with the Crashlytics module in the cloud (or using Create Deployment Package - versioned) in Studio Pro 9.10.1, it gives an error when building the native packager.

The error is
```
C:\Program Files\Mendix\9.10.1.37625\modeler\tools\node\node.exe C:\Program Files\Mendix\9.10.1.37625\modeler\tools\node\node_modules\react-native\local-cli\cli.js bundle --config "C:\Users\Arjen.Lammers\AppData\Local\Temp\1n3hob0n.fx1\deployment\native\metro.config.json" --dev "false" --entry-file "C:\Users\Arjen.Lammers\AppData\Local\Temp\1n3hob0n.fx1\deployment\native\index.js" --platform "ios" --bundle-output "C:\Users\Arjen.Lammers\AppData\Local\Temp\1n3hob0n.fx1\deployment\native\bundle\iOS\index.ios.bundle" --assets-dest "C:\Users\Arjen.Lammers\AppData\Local\Temp\1n3hob0n.fx1\deployment\native\bundle\iOS"
[Output]                     Welcome to Metro!
[Output]               Fast - Scalable - Integrated


[Error] error Unable to resolve module ../../../themesource/crashlytics/node_modules/@react-native-firebase/crashlytics from C:\Users\Arjen.Lammers\AppData\Local\Temp\1n3hob0n.fx1\javascriptsource\crashlytics\actions\SetUserId.js: 

[Error] None of these files exist:
[Error]   * ..\..\themesource\crashlytics\node_modules\@react-native-firebase\crashlytics(.native|.ios.js|.native.js|.js|.ios.jsx|.native.jsx|.jsx|.ios.ts|.native.ts|.ts|.ios.tsx|.native.tsx|.tsx|.ios.cjs|.native.cjs|.cjs|.ios.mjs|.native.mjs|.mjs|.ios.json|.native.json|.json|.ios.js_commonjs-exports|.native.js_commonjs-exports|.js_commonjs-exports|.ios.js_commonjs-module|.native.js_commonjs-module|.js_commonjs-module)
[Error]   * ..\..\themesource\crashlytics\node_modules\@react-native-firebase\crashlytics\index(.native|.ios.js|.native.js|.js|.ios.jsx|.native.jsx|.jsx|.ios.ts|.native.ts|.ts|.ios.tsx|.native.tsx|.tsx|.ios.cjs|.native.cjs|.cjs|.ios.mjs|.native.mjs|.mjs|.ios.json|.native.json|.json|.ios.js_commonjs-exports|.native.js_commonjs-exports|.js_commonjs-exports|.ios.js_commonjs-module|.native.js_commonjs-module|.js_commonjs-module)
[Error] [0m [90m  7 |[39m [90m// Other code you write will be lost the next time you deploy the project.[39m[0m
[Error] [0m [90m  8 |[39m [36mimport[39m { [33mBig[39m } [36mfrom[39m [32m"big.js"[39m[33m;[39m[0m
[Error] [0m[31m[1m>[22m[39m[90m  9 |[39m [36mimport[39m crashlytics [36mfrom[39m [32m'../../../themesource/crashlytics/node_modules/@react-native-firebase/crashlytics'[39m[33m;[39m[0m
[Error] [0m [90m    |[39m                          [31m[1m^[22m[39m[0m
[Error] [0m [90m 10 |[39m[0m
[Error] [0m [90m 11 |[39m [90m// BEGIN EXTRA CODE[39m[0m
[Error] [0m [90m 12 |[39m [90m// END EXTRA CODE[39m[0m.
[Output] Error: Unable to resolve module ../../../themesource/crashlytics/node_modules/@react-native-firebase/crashlytics from C:\Users\Arjen.Lammers\AppData\Local\Temp\1n3hob0n.fx1\javascriptsource\crashlytics\actions\SetUserId.js: 

[Output] None of these files exist:
[Output]   * ..\..\themesource\crashlytics\node_modules\@react-native-firebase\crashlytics(.native|.ios.js|.native.js|.js|.ios.jsx|.native.jsx|.jsx|.ios.ts|.native.ts|.ts|.ios.tsx|.native.tsx|.tsx|.ios.cjs|.native.cjs|.cjs|.ios.mjs|.native.mjs|.mjs|.ios.json|.native.json|.json|.ios.js_commonjs-exports|.native.js_commonjs-exports|.js_commonjs-exports|.ios.js_commonjs-module|.native.js_commonjs-module|.js_commonjs-module)
[Output]   * ..\..\themesource\crashlytics\node_modules\@react-native-firebase\crashlytics\index(.native|.ios.js|.native.js|.js|.ios.jsx|.native.jsx|.jsx|.ios.ts|.native.ts|.ts|.ios.tsx|.native.tsx|.tsx|.ios.cjs|.native.cjs|.cjs|.ios.mjs|.native.mjs|.mjs|.ios.json|.native.json|.json|.ios.js_commonjs-exports|.native.js_commonjs-exports|.js_commonjs-exports|.ios.js_commonjs-module|.native.js_commonjs-module|.js_commonjs-module)
[Output] [0m [90m  7 |[39m [90m// Other code you write will be lost the next time you deploy the project.[39m[0m
[Output] [0m [90m  8 |[39m [36mimport[39m { [33mBig[39m } [36mfrom[39m [32m"big.js"[39m[33m;[39m[0m
[Output] [0m[31m[1m>[22m[39m[90m  9 |[39m [36mimport[39m crashlytics [36mfrom[39m [32m'../../../themesource/crashlytics/node_modules/@react-native-firebase/crashlytics'[39m[33m;[39m[0m
[Output] [0m [90m    |[39m                          [31m[1m^[22m[39m[0m
[Output] [0m [90m 10 |[39m[0m
[Output] [0m [90m 11 |[39m [90m// BEGIN EXTRA CODE[39m[0m
[Output] [0m [90m 12 |[39m [90m// END EXTRA CODE[39m[0m
[Output]     at ModuleResolver.resolveDependency (C:\Program Files\Mendix\9.10.1.37625\modeler\tools\node\node_modules\metro\src\node-haste\DependencyGraph\ModuleResolution.js:211:15)
[Output]     at DependencyGraph.resolveDependency (C:\Program Files\Mendix\9.10.1.37625\modeler\tools\node\node_modules\metro\src\node-haste\DependencyGraph.js:413:43)
[Output]     at Object.resolve (C:\Program Files\Mendix\9.10.1.37625\modeler\tools\node\node_modules\metro\src\lib\transformHelpers.js:317:42)
[Output]     at resolve (C:\Program Files\Mendix\9.10.1.37625\modeler\tools\node\node_modules\metro\src\DeltaBundler\traverseDependencies.js:629:33)
[Output]     at C:\Program Files\Mendix\9.10.1.37625\modeler\tools\node\node_modules\metro\src\DeltaBundler\traverseDependencies.js:645:26
[Output]     at Array.reduce (<anonymous>)
[Output]     at resolveDependencies (C:\Program Files\Mendix\9.10.1.37625\modeler\tools\node\node_modules\metro\src\DeltaBundler\traverseDependencies.js:644:33)
[Output]     at C:\Program Files\Mendix\9.10.1.37625\modeler\tools\node\node_modules\metro\src\DeltaBundler\traverseDependencies.js:329:33
[Output]     at Generator.next (<anonymous>)
[Output]     at asyncGeneratorStep (C:\Program Files\Mendix\9.10.1.37625\modeler\tools\node\node_modules\metro\src\DeltaBundler\traverseDependencies.js:137:24)
[Output] info Run CLI with --verbose flag for more details.
```
Also re-imported the module, but it made no difference.

**Comment by cdcharlebois** [comment]
Out of curiosity, does it work if you run ‘npm install’ in the ‘themesource/crashlytics’ directory?

**Comment by cdcharlebois** [comment]
fixed in 2.0.0
